cedar-policy/src/ffi/convert.rs

```diff
@@ -225,6 +225,23 @@ pub enum SchemaToJsonAnswer {
     },
 }
 
+/// Convert `cedar_policy::ffi::PolicySet` JSON to `cedar_policy::PolicySet` Object
+pub fn json_to_policyset(
```

Comment:
Elsewhere (e.g., in `ffi/is_authorized.rs`), function names with just `json` take a `serde_json::Value`, while functions like this that take a string have names with `json_str`